pkg/controller/networkpolicy/validate.go

```diff
@@ -175,7 +173,7 @@ func (v *NetworkPolicyValidator) Validate(ar *admv1.AdmissionReview) *admv1.Admi
 		msg, allowed = v.validateTier(&curTier, &oldTier, op, ui)
 	case "ClusterGroup":
 		klog.V(2).Info("Validating ClusterGroup CRD")
-		var curCG, oldCG crdv1alpha2.ClusterGroup
+		var curCG, oldCG crdv1beta1.ClusterGroup
```

Comment:
Just to confirm: if users use a previous version to create the resource, it will be converted to the stored version before calling the webhook so we don't need to handle crdv1alpha1, right?

Reply:
No, in the validation webhook, it will still be the version used to create it. v1alpha3 and v1beta1 have the same structure and validating logic, we could unmarshal both of them into a v1beta1 object to do the validation. And after all we only store the v1beta1 in the etcd.
Actually, here is a legacy issue. As you can see, we were still using v1alpha2 here to do validation, which is no longer served. But v1alpha2 has more fields than v1alpha3, both of them could be unmarshal into v1alpha2 object. we haven't met any problems.